packages/commcare/test/index.js

```diff
@@ -159,21 +157,21 @@ describe('getCases', () => {
       },
     };
 
-    const { data, response } = await execute(get('case'))(state);
+    const { data, response } = await execute(
+      get('case', { limit: 1, offset: 0 })
+    )(state);
 
     expect(data.length).to.equal(1);
     expect(data[0]).to.haveOwnProperty('case_id');
     expect(response.meta.limit).to.equal(1);
   });
-
   it('should fetch a single case', async () => {
     testServer
       .intercept({
-        path: `/a/${domain}/api/v0.5/case/12345`,
+        path: /\/a\/my-domain\/api\/v0\.5\/case\/12345/,
```

Comment:
We shouldn't need a regex here really. In fact, when requesting a single case, we shouldn't add `offset` and `limit` to the URL at all.

This test has alerted me that there's something amiss in the implementation, and we need think about our strategy a little harder.